cmd/kubeadm/app/cmd/phases/kubeconfig.go

```diff
@@ -116,7 +125,14 @@ func runKubeConfigFile(kubeConfigFileName string) func(workflow.RunData) error {
 
 		// if external CA mode, skip certificate authority generation
 		if data.ExternalCA() {
-			//TODO: implement validation of existing kubeconfig files
+			config, err := getKubeConfig(data.Cfg(), kubeConfigFileName)
```

Comment:
Is this new behavior covered either directly or indirectly in the _tests file?  

/assign @fabriziopandini

Reply:
Not yet, I'll update the tests tomorrow.